### PR TITLE
[it] tutorial-keras-overfit_and_underfit

### PR DESCRIPTION
**Pull request by codeadmin-peritiae**
Basic translation

### CODE REVIEW

File: site/it/tutorials/keras/overfit_and_underfit.ipynb [hunk 1]
```diff
@@ -913,7 +913,7 @@
         "id": "Fy3CMUZpzH3d"
       },
       "source": [
-        "### Traccia dele perdite in addestramento e validazione"
+        "### Traccia delle funzioni obiettivo in addestramento e validazione"
```
Comment (bhack) [inline site/it/tutorials/keras/overfit_and_underfit.ipynb:916]: or `Tracciato`?

Comment (codeadmin-peritiae) [reply site/it/tutorials/keras/overfit_and_underfit.ipynb:916]: "Andamento" is probably better. Thank you

**Comment by codeadmin-peritiae** [comment]
> #### Preview
> Preview and run these notebook edits with Google Colab:
> * [site/it/tutorials/keras/overfit_and_underfit.ipynb](https://colab.research.google.com/github/codeadmin-peritiae/docs-l10n/blob/tutorial-keras-overfit_and_underfit/site/it/tutorials/keras/overfit_and_underfit.ipynb)
> 
> #### Diffs
> For cleaner diffs (and easier review), format notebooks with the [nbfmt.py](https://github.com/tensorflow/docs/blob/master/tools/nbfmt.py) tool:
> `./tools/nbfmt.py --ignore_warn path/to/notebook.ipynb`
> Rendered [notebook diffs](https://app.reviewnb.com/tensorflow/docs-l10n/pull/172/files/) available on ReviewNB.com.

@lamberta
 
I tryed to check this notebook with nbfmt.py but I get an error (please see below):

xxx@Gianlucas-Mac tools % ./nbfmt.py site/it/tutorials/keras/overfit_and_underfit.ipynb

  File "./nbfmt.py", line 55
    print(f" \033[33m {msg}\033[00m", file=sys.stderr)
                                   ^
SyntaxError: invalid syntax

### **Syntax error ???**

Any suggestion?
Thank you.

**Comment by lamberta** [comment]
Hmm. Try python3 ./nbfmt.py ?

**Comment by codeadmin-peritiae** [comment]
Switch to different repository

**Pull request closed without merging**